docs/changelog.rst

```diff
@@ -12,6 +12,7 @@ Changelog
 Added
 ~~~~~
 - ``generate_tracking_link`` method for ``Offer`` model. (`iamanikeev`_)
+- Support of Tag controller. (`iamanikeev`_)
```

Comment:
Don't add new changes to an existing release. It doesn't make sense. Please, add it to an appropriate section

Reply:
Sorry, this is some kind of optical illusion in my brain, I always miss the latest release section and think of latest `Added` section a part of `Unreleased`. Fixed.